packages/integration-tests/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sentry-browser-integration-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rimraf -g suites/**/dist",
+    "install-browsers": "playwright install --with-deps",
+    "lint": "run-s lint:prettier lint:eslint",
+    "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
+    "lint:prettier": "prettier --check \"{suites,utils}/**/*.ts\"",
+    "test:ci": "playwright test ./suites --browser='all' --reporter='line'",
+    "test": "playwright test ./suites",
+    "posttest": "yarn clean"
+  },
+  "dependencies": {
+    "@playwright/test": "^1.17.0",
+    "@sentry/browser": "file:../browser",
+    "babel-loader": "^8.2.2",
+    "handlebars-loader": "^1.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "playwright": "^1.17.1",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.5.2",
+    "webpack": "^5.52.0"
+  },
+  "resolutions": {
+    "@sentry/core": "file:./packages/core",
+    "@sentry/hub": "file:./packages/hub",
+    "@sentry/integrations": "file:./packages/integrations",
+    "@sentry/minimal": "file:./packages/minimal",
+    "@sentry/tracing": "file:./packages/tracing",
+    "@sentry/types": "file:./packages/types",
+    "@sentry/utils": "file:./packages/utils"
```

Comment:
Reminder to self:

Maybe we could solve this on the Webpack level -- we do something similar in https://github.com/getsentry/sentry-javascript/blob/7291695de01209719e903533ca771f7f1d621e87/scenarios/browser/webpack.js#L45-L60

Reply:
@rhcarvalho, yes that works well -> 96a844c